Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/OfferingList/APIKeyDashboardList.swift

```diff
@@ -200,6 +203,15 @@ struct APIKeyDashboardList: View {
                     }
                 }
         }
+        .overlay {
+            if offeringToPresent != nil {
+                VStack {}.presentPaywallIfNeeded(offering: offeringToPresent,
```

Comment:
Is there a better way to get a paywall to conditionally appear? I was trying to get something working similar to the `sheet` modifier that would make the paywall appear whenever the offering was not nil, but couldn't come up with anything. This feels convoluted.

Reply:
That's a lot nicer, I'll make this change!